### ISSUE TITLE
Health Care Rewriting: VHIC

### ISSUE THREAD

**Issue by Karak888** [department-of-veterans-affairs/va.gov-team#2542]
@DanielleSoCompany commented on [Fri Feb 15 2019](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/16957)

## User story

As a Veteran, I want to understand the different ID cards I may need, including the Veterans Health ID card.

## What needs to be done

This page (https://www.va.gov/healthbenefits/vhic/index.asp) is currently linked to from “Types of Veteran ID Cards” (https://www.va.gov/records/get-veteran-id-cards/). Rewrite/recreate page to align with plain language/VA.gov style for consistency of benefit information experience.

### Steps to completion

- [x] Finalize IA recommendations @mnorthuis @DanielleSoCompany 
- [x] First draft: Rewrite page to align with plain language and VA.gov style standards, as well as any additional content revisions needed to accomodate IA changes. @AnneHurley 
- [x] Copy edit and build page in Github @RLHecht (with support from @bethpotts) 
- [x] Strategy/voice edit @DanielleSoCompany
- [x] Build page in CMS @lillianjwarner  
- [x] Final copy edit and create PDF of page for SME review @RLHecht 
- [x] Send PDF to SMEs for review @DanielleSoCompany 
- [x] SME approval received
- [x] Incorporate SME edits @DanielleSoCompany 
- [x] Implement SME edits in CMS and publish page @RLHecht 
- [ ] Validate on live site 

### Writing instructions and resources

Create a new page for "Request a Veterans Health ID Card (VHIC)". This page will sit under the "Manage" spoke within the Health Care hub.

**Source document:** https://www.va.gov/records/get-veteran-id-cards/

**Keywords:** vhic, veterans health identification card

**WIP url:** /health-care/request-health-id-card/

---

@AnneHurley commented on [Fri Mar 29 2019](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/16957#issuecomment-478089756)

[VHIC Cards v1 for copyedit]
(https://app.zenhub.com/files/31788863/5a1eea0b-bd40-4a45-b364-7044b9301723/download)

---

@AnneHurley commented on [Fri Mar 29 2019](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/16957#issuecomment-478090143)

@RLHecht and @DanielleSoCompany and @bethpotts - can you see the uploaded doc above?

---

@RLHecht commented on [Fri Mar 29 2019](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/16957#issuecomment-478093364)

@AnneHurley I can.

---

@AnneHurley commented on [Tue Apr 02 2019](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/16957#issuecomment-479113332)

@RLHecht and @bethpotts - here are three related resources for the bottom of this page:

More information and resources

Other Veteran ID cards > link https://www.va.gov/records/get-veteran-id-cards/ 
Learn about the other types of identification cards you can get to show you’re a Veteran. 

Get VA health care > link https://www.va.gov/health-care/
Learn how to apply for VA health care benefits, manage your appointments and benefits, and more. 

VA health care costs   > https://www.va.gov/healthbenefits/cost/index.asp 
Learn about your copay, financial assessment, and how VA health care works with private health insurance.

Note - I've made the overall heading and each headline sentence case 

---

@bethpotts commented on [Tue Apr 02 2019](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/16957#issuecomment-479191625)

@RLHecht You'll insert these in the meta date of the file. We can do them together on Wednesday.

---

@RLHecht commented on [Tue Apr 09 2019](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/16957#issuecomment-481340212)

@DanielleSoCompany This page is ready for you to look at. 

---

@mnorthuis commented on [Tue Apr 16 2019](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/16957#issuecomment-483818609)

Confirming URL for this page - it should be `/health-care/request-health-id-card/`

@DanielleSoCompany is there a reason we went with "request" language versus "get" for this one?  I just wanted to confirm...

---

@DanielleSoCompany commented on [Wed Jun 05 2019](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/16957#issuecomment-499287091)

@Karak888 FYI - Moved this into In Progress again because I've started my review/edit. Thanks!

---

@DanielleSoCompany commented on [Wed Jun 05 2019](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/16957#issuecomment-499297415)

Hi @RLHecht. Would you please give this a final copy edit when you have the time? Not a huge rush. (**Note:** Do not merge. This is just for SME preview/approval). Thank you!

**Github file:** https://github.com/department-of-veterans-affairs/vagov-content/blob/content/wip/VHIC/pages/health-care/get-health-id-card.md

**Heroku link:** https://vagov-content-pr-360.herokuapp.com/health-care/get-health-id-card/

**Full PR:** https://github.com/department-of-veterans-affairs/vagov-content/pull/360

---

@DanielleSoCompany commented on [Wed Jun 05 2019](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/16957#issuecomment-499297751)

@mnorthuis slacking you re: title

---

@DanielleSoCompany commented on [Wed Jun 05 2019](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/16957#issuecomment-499301960)

Hi again, @RLHecht. I think we're going to have to figure out a different way to preview these pages with stakeholders. Once I merged master into this branch, none of the health care pages are working (I'm assuming because they're not being handled in CMS). 

We may need to use Word docs or a combination of Word docs and screenshots of draft pages in CMS. Let's connect on this this week. Thanks!

---

@mnorthuis commented on [Thu Jun 06 2019](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/16957#issuecomment-499460829)

Please update URL to `/health-care/get-health-id-card/` for consistency with other pages.  

---

@DanielleSoCompany commented on [Thu Jun 06 2019](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/16957#issuecomment-499475468)

@mnorthuis Done - thanks!

---

@DanielleSoCompany commented on [Thu Jun 06 2019](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/16957#issuecomment-499504673)

@RLHecht @lillianjwarner I've adjusted the steps to completion above to reflect new process involving CMS. Thanks!

---

@lillianjwarner commented on [Thu Jun 06 2019](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/16957#issuecomment-499582996)

@DanielleSoCompany this page is built in CMS.

I'm wondering if this text: "_After you've received your VA health care welcome call_" should be an h4 instead of an h3? The Q/A titles are H3s and their size can't be changed.

---

@DanielleSoCompany commented on [Thu Jun 06 2019](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/16957#issuecomment-499678445)

Thank you, @lillianjwarner! Yes, I think that makes sense. Thank you!
 

---

@DanielleSoCompany commented on [Tue Jun 18 2019](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/16957#issuecomment-503320865)

Hi @lillianjwarner. Could you direct @RLHecht to this page within the CMS so that she can give it a final proof? Then, could you create a PDF of the page so we can send to SMEs for review? Thank you!!

---

@RLHecht commented on [Tue Jun 18 2019](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/16957#issuecomment-503355833)

@DanielleSoCompany I did a final proofread of this page in CMS and all looks good. No changes. I emailed you the PDF from my va.gov account. 

---

@DanielleSoCompany commented on [Tue Jun 25 2019](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/16957#issuecomment-505409369)

PDF sent to SMEs for review.
[Brand Consolidation Rewriting_Veteran Health Identification Card_06.24.19](https://app.zenhub.com/files/31788863/a3a35320-dc4e-4aa3-b724-dfc9eafdab21/download)

SMEs: Robin Brown, Jeff Grandon (VHA)

@Karak888 For awareness. As discussed, I'm moving issues to backlog and adding stakeholder labels once we've sent to SMEs for review. These can be moved back to In-Progress when there's feedback to implement. 

---

@DanielleSoCompany commented on [Tue Jul 09 2019](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/16957#issuecomment-509823993)

@Karak888 We've received SME feedback. Will be trying to implement/push live this week. Moving to In Progress.

---

@Karak888 commented on [Wed Jul 10 2019](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/16957#issuecomment-510138546)

Sounds Great!!! thanks @DanielleSoCompany 

---

@DanielleSoCompany commented on [Sat Jul 13 2019](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/16957#issuecomment-511151576)

SME feedback incorporated and page updated/pushed to production in CMS.

Revised copy, based on SME feedback around verifying identity for new card:

**What should I do if my VHIC is lost or stolen?**
You can request a replacement card at the VA medical center where you had your picture taken. You’ll need to bring your driver’s license, passport, or other government ID or be able to answer a series of challenge questions so we can verify your identity before we give you a new card.

If you’ve changed your medical center since getting your original card, you can contact the medical center where you currently receive care. Or, call us at 877-222-8387, Monday through Friday, 8:00 a.m. to 8:00 p.m. ET. Our staff will need to verify your identity with a series of challenge questions before we can send a new card.

---

@RLHecht commented on [Thu Aug 15 2019](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/16957#issuecomment-521655550)

@Karak888 This ticket can be closed. Just confirmed the most recent SME update is live on the site. 

---

@mnorthuis commented on [Mon Oct 14 2019](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/16957#issuecomment-541906357)

@RLHecht @Karak888 I don't see this page live on the site?

---

@RLHecht commented on [Tue Oct 15 2019](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/16957#issuecomment-542245592)

@mnorthuis Just found the page: https://www.va.gov/get-a-veteran-health-identification-card-vhic/ It appears to have "-vhic" at the end of the URL. Not sure why that is. Should I change it so it is just "get-a-veteran-health-identification-card" or should be "get-health-id-card" like listed above? 

---

@mnorthuis commented on [Tue Oct 15 2019](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/16957#issuecomment-542258359)

That is definitely not the correct URL and appears to be using the H1.   It should be `get-health-id-card`. 
It also looks like the hub and left nav were never updated in support of the launch of this page. 

---

@mnorthuis commented on [Tue Oct 15 2019](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/16957#issuecomment-542348919)

Corrections to be made: 

this page: `/get-a-veteran-health-identification-card-vhic/`
should be: `/health-care/get-health-id-card/`

The URL is incorrect and it should live with Health Care, not at the root level. 

In addition, links to this new page need to be updated, they are still pointing at the legacy page:
- in the health care left nav, under Manage Benefits spoke
- on the https://www.va.gov/records/get-veteran-id-cards/ page

---

@RLHecht commented on [Wed Oct 16 2019](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/16957#issuecomment-542743272)

@jenniferlee-dsva Can you please check my work in Drupal before today's deployment. I did the following:
- Changed the URL for the page https://www.va.gov/get-a-veteran-health-identification-card-vhic/ to /health-care/get-health-id-card/
- I checked the "Provide a Menu Link" box
- I moved the page under Manage Benefits in the Health Care hub and disabled the legacy page that was there in its place.
- I updated the link on this records page: https://www.va.gov/records/get-veteran-id-cards/ so it would point to the new page and not the legacy page. That's saved as a Draft so will need to be Published

Drupal wouldn't let me save the URL and menu work I did on the page in a draft state, so it's currently as Published, which is why I would like to confirm all is good before today's deployment. 

FYI @mnorthuis 

---

@jenniferlee-dsva commented on [Wed Oct 16 2019](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/16957#issuecomment-542758185)

I already see that it is live, and it looks all right. (Did you mean for it to not be live yet?)

---

@jenniferlee-dsva commented on [Wed Oct 16 2019](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/16957#issuecomment-542760162)

This part isn't live yet but in CMS preview, it takes me to the new page, so all looks good. 

•	I updated the link on this records page: https://www.va.gov/records/get-veteran-id-cards/ so it would point to the new page and not the legacy page

---

@RLHecht commented on [Wed Oct 16 2019](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/16957#issuecomment-542765043)

@jenniferlee-dsva I didn't mean it to be live but I guess that works! Drupal wasn't allowing me to save as a draft only as published but I assumed it wouldn't be live until today's deployment. Thanks for checking!

---

@RLHecht commented on [Wed Oct 16 2019](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/16957#issuecomment-542766722)

@mnorthuis This work is now done. I'll validate the updated link on the records page once it's live but the VHIC page is now live with the correct URL. @Karak888

**Comment by mnorthuis** [comment]
This is duplicate of #2579
